### PR TITLE
Flag for enabling unit test processing

### PR DESCRIPTION
**Pull request by octa-one**
Hi 👋
In our project, we want to run unit tests on classes collected with Colonist. The main idea is to check navigation destinations.
It would be convenient to have a "processTest" flag in the android plugin, similar to the java plugin.
So I added it in this pull request.

### CODE REVIEW

**Comment by dkostyrev** [comment]
Thanks for your contribution! Could you please enable this flag in `:colonist-samples:modular-android:modular-app` and fix `FeatureManagerTest`? For reference, it was fixed in [this commit](https://github.com/joomcode/colonist/commit/ac3c012f3337f95af243f2b724d1a7eb4029124c).

**Comment by octa-one** [comment]
> Thanks for your contribution! Could you please enable this flag in `:colonist-samples:modular-android:modular-app` and fix `FeatureManagerTest`? For reference, it was fixed in [this commit](https://github.com/joomcode/colonist/commit/ac3c012f3337f95af243f2b724d1a7eb4029124c).

Done!

File: colonist-samples/modular-android/modular-app/src/test/java/com/joom/colonist/modular/FeatureManagerTest.kt [hunk 1]
```diff
@@ -16,13 +16,21 @@
...
-import com.joom.colonist.ColonistException
+import com.joom.colonist.modular.feature1.Feature1
+import com.joom.colonist.modular.feature2.Feature2
+import org.junit.Assert
 import org.junit.Test
 
 class FeatureManagerTest {
-  @Test(expected = ColonistException::class)
+  @Test
   fun testFeatureManagerIsNotProcessed() {
```
Comment (dkostyrev) [inline colonist-samples/modular-android/modular-app/src/test/java/com/joom/colonist/modular/FeatureManagerTest.kt]: I guess it should be `testFeatureManagerIsProcessed` in this case

**Review by dkostyrev — APPROVED** [review]

File: colonist-samples/modular-android/modular-app/src/test/java/com/joom/colonist/modular/FeatureManagerTest.kt (hunk 1 above)
Comment (octa-one) [reply colonist-samples/modular-android/modular-app/src/test/java/com/joom/colonist/modular/FeatureManagerTest.kt]: Sure

**Comment by dkostyrev** [comment]
Thanks! I will fork your branch in this repository and reopen pull request to run checks.

**Comment by dkostyrev** [comment]
I've published a new release on maven central. It should be available in a couple of hours.

**Pull request merged**